import & export configuration

tetuaoro opened the pull request:
## Description
This pull request adds import and export configuration features to `wg-easy`. Users can now easily save and load their configurations from the user interface.

## Changes
- Added import configuration feature.
- Added export configuration feature.
- Updated the user interface to include new import/export buttons.

Thank you for taking the time to review these changes ! Let me know if you have any feedback.

pheiduck commented:
Other things looks fine.
The new function must be translated into the other languages we support.

tetuaoro commented:
Hi,

I've added translations for French, Spanish, and Italian, and updated the terminology to use "restore" instead of "import" and "backup" instead of "export" for better clarity and relevance. Additionally, I renamed the relevant functions and updated the README with a new screenshot to reflect these changes.

![screenshot](https://github.com/wg-easy/wg-easy/assets/65575727/7a79d08b-f50f-4739-9074-5a2a1ac10fa6)

pheiduck commented:
@tetuaoro as a bonus I added german translation to it.

tetuaoro commented on src/lib/Server.js:
```diff
@@ -276,7 +293,7 @@ module.exports = class Server {
             const filePath = safePathJoin(publicDir, id);
 
-            const stats = await stat(filePath).catch(() => {});
+            const stats = await stat(filePath).catch(() => { });
```
Ah, maybe it's the `formatter` vscode extension or the `eslint`, idk.

pheiduck commented:
Login is not possible with that, needs rework

tetuaoro commented:
> Login is not possible with that, needs rework

How can I test that ? I use `--env PASSWORD=...` and I can login & logout.

pheiduck commented:
reopen the PR, then we can make a development image first
